ksql-engine/src/main/java/io/confluent/ksql/engine/SqlFormatInjector.java

```diff
@@ -47,7 +47,7 @@ public <T extends Statement> ConfiguredStatement<T> inject(
       return statement.withStatement(sql, (T) prepare.getStatement());
     } catch (final Exception e) {
       throw new KsqlException("Unable to format statement! This is bad because "
-          + "it means we cannot persist it onto the command topic: " + statement, e);
+          + "it means we cannot persist it onto the command topic ", e);
```

Comment:
This change not needed on newer versions (starting with 7.0.x), right? It'd be good to keep the more informative error message wherever we can, on newer branches.